cube/CMakeLists.txt

```diff
@@ -199,7 +199,7 @@ if(APPLE)
     # Fix up the library references to be self-contained within the bundle.
     install(CODE "
         include(BundleUtilities)
-        fixup_bundle(\${CMAKE_INSTALL_PREFIX}/cube/cube.app \"\" \"\")
+        fixup_bundle(\${CMAKE_INSTALL_PREFIX}/cube/cube.app \"\" ${Vulkan_LIBRARY_DIR})
```

Comment:
I think you need to keep the (escaped) double-quotes here.  If the path contains a space, the path may get interpreted as two arguments by `fixup_bundle()`.   I know that CMake usually handles this pretty well in most cases, but here, under the `CODE` property and the need to escape the quotes, I'm not so sure.

You can check this pretty easily by creating a dir called `/tmp/xxx xxx`, copying your entire build tree there, and then re-run CMake and rebuild.  But I wouldn't bother and just add the quotes back.